### PR TITLE
Add ability to modify the HostBuilder before building it. Adding sample for altering HostBuilder slightly for multiple commands.

### PR DESCRIPTION
**Pull request by ejsmith**

### CODE REVIEW

File: src/Oakton/CommandLineHostingExtensions.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public static class CommandLineHostingExtensions
         /// <returns></returns>
         public static Task<int> RunOaktonCommands(this IHostBuilder builder, string[] args)
         {
-            return Execute(builder, null, args);
+            return Execute(builder, Assembly.GetEntryAssembly(), args);
```
Comment (ejsmith) [inline src/Oakton/CommandLineHostingExtensions.cs:26]: @jeremydmiller not sure how you feel about this change. I didn't see how else to get commands that are declared in the application assembly to be added when using the `RunOaktonCommands` extension.

Comment (jeremydmiller) [reply src/Oakton/CommandLineHostingExtensions.cs:26]: That's a good oversight to get fixed, yeah. Used to depend on walking up the call stack, but that doesn't play nicely in every circumstance so I ditched it. I agree with that, but I'll need to change the docs a bit.

Comment (ejsmith) [reply src/Oakton/CommandLineHostingExtensions.cs:26]: Hmm... thinking about it a bit more. What if my commands are not defined in the web application entry assembly? Seems like there needs to be some overload or optional parameters on `RunOaktonCommands` to add individual assemblies or tell it to scan all assemblies. But maybe that is what `OaktonCommandAssemblyAttribute` is for, eh?

Comment (jeremydmiller) [reply src/Oakton/CommandLineHostingExtensions.cs:26]: That's what `OaktonCommandAssemblyAttribute` is for:) Sorry for the delay on this one, I'm getting in and released with Oakton 3.1 today.

**Comment by ejsmith** [comment]
Thank you.

**Pull request merged**